app/commands/v2/import.rb

```diff
@@ -0,0 +1,134 @@
+module Commands
+  module V2
+    class Import < BaseCommand
+
+      def self.call(payload)
+        logger.debug "#{self} called with payload:\n#{payload}"
+
+        response = ContentItem.transaction do
+          PublishingAPI.service(:statsd).time(self.name.gsub(/:+/, '.')) do
+            new(payload, event: nil, downstream: true, nested: false, callbacks: []).call
+          end
+        end
+
+        response
+      end
+
+      def call
+        unless UuidValidator.valid?(payload[:content_id])
+          raise CommandError.new(
+          code: 422,
+          error_details: {
+            error: {
+              code: 422,
+              message: "Content id not valid",
+              fields: "content_id",
+            }
+          })
+        end
+
+        delete_all(payload[:content_id])
+        all_content_items.map.with_index do |event, index|
+          create_content_item(event, index, payload[:content_id])
+        end
+
+        send_downstream(all_content_items.find { |e| e[:action] == 'Publish' })
```

Comment:
this will need to be wrapped in after_transaction_commit like https://github.com/alphagov/publishing-api/blob/master/app/commands/v2/put_content.rb#L27